### PR TITLE
Request a user-defined number of comments and refresh

### PR DESCRIPTION
**Pull request by cgregori**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -35,9 +35,57 @@ async function helloName() {
   });
 }
 
+/** Fetches comments from the server and adds them to the DOM*/
```
Comment (theannielin) [inline portfolio/src/main/webapp/script.js:38]: grammar nit: period and space at end of somment

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -35,9 +35,57 @@ async function helloName() {
...
+  });
+}
+
+function parseMaxComments() {
+  let currentUrl = getURL();
+  let parser = new URL(currentUrl); 
+  let queryString = parser.search;
+  let pos = queryString.search("maxComments=");
+  let maxComments
+  if(pos != -1) {
```
Comment (theannielin) [inline portfolio/src/main/webapp/script.js:56]: Could you comment what this is doing? It's a bit unclear why 12 would be the number here. Setting default max comments as a constant value would also help clarify this.

Comment (cgregori) [reply portfolio/src/main/webapp/script.js:56]: I add 12 b/c the length of "maxComments=" is 12, and pos returns the beginning index of "maxComments=". I'll add a clarifying comment

Comment (theannielin) [reply portfolio/src/main/webapp/script.js:56]: Yeah, please add some clarification.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -54,8 +54,10 @@ function parseMaxComments() {
   let pos = queryString.search("maxComments=");
   let maxComments
   if(pos != -1) {
+    //Add "maxComments=".length (12) to position
      maxComments = queryString.slice(pos+12);
   } else {
+    //Default value of maxComments
```
Comment (theannielin) [inline portfolio/src/main/webapp/script.js:60]: nit: space after // (same for above)

Comment (cgregori) [reply portfolio/src/main/webapp/script.js:60]: Ah, I see

**Pull request merged**